patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -131,7 +131,15 @@
      }
  
      public boolean func_70093_af()
-@@ -2092,7 +2146,7 @@
+@@ -2047,6 +2101,7 @@
+ 
+     public void func_71027_c(int p_71027_1_)
+     {
++        if (!net.minecraftforge.common.ForgeHooks.onTravelToDimension(this, p_71027_1_)) return;
+         if (!this.field_70170_p.field_72995_K && !this.field_70128_L)
```

Comment:
Move it inside this condition, we dont want it firing client side {sync issues} and dont want it firing when the entity wouldnt move {cuz dumb}